src/github.com/getlantern/launcher/launcher_windows.go

```diff
@@ -0,0 +1,38 @@
+// Package launcher configures Lantern to run on system start
+package launcher
+
+import (
+	"github.com/kardianos/osext"
+	"github.com/luisiturrios/gowin"
+
+	"github.com/getlantern/golog"
+)
+
+const (
+	RunDir = `Software\Microsoft\Windows\CurrentVersion\Run`
```

Comment:
Yeah, true. I usually prefer declaring constants at the top of a file regardless, but I went ahead and moved it inside the function body.